js/conversation_controller.js

```diff
@@ -207,6 +212,44 @@
 
       return conversation;
     },
+    async deleteContact(id, type) {
```

Comment:
is `type` used anywhere?

Reply:
Right yeah don't think this is required, just copied from another function